src/ui/card/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react';
+import { Card as MUICard, Paper } from '@mui/material';
+import './_card.scss';
+
+type CardPropType = {
+  className?: string;
+  children: ReactElement;
+  backgroundColor?: string;
+  elevation?: number;
+  shadow?: boolean;
+  transparent?: boolean;
+};
+
+const Card = ({
+  className = '',
```

Comment:
<img width="283" alt="image" src="https://user-images.githubusercontent.com/25376645/165337506-e4ee50d5-8803-4f2b-a2c5-109dca1fc323.png">

transparent not working